v19.2/verify-constraint-conformance.md

```diff
@@ -0,0 +1,565 @@
+       55 | PARTITION europe_west OF INDEX movr.public.rides@primary                                       | NULL       | movr          | rides      | primary                                       |                       9  
+       55 | PARTITION europe_west OF INDEX movr.public.rides@primary                                       | NULL       | movr          | rides      | primary                                       |                       9  
+       55 | PARTITION europe_west OF INDEX movr.public.rides@rides_auto_index_fk_city_ref_users            | NULL       | movr          | rides      | rides_auto_index_fk_city_ref_users            |                       9  
+       55 | PARTITION europe_west OF INDEX movr.public.rides@rides_auto_index_fk_city_ref_users            | NULL       | movr          | rides      | rides_auto_index_fk_city_ref_users            |                       9  
+       55 | PARTITION europe_west OF INDEX movr.public.rides@rides_auto_index_fk_city_ref_users            | NULL       | movr          | rides      | rides_auto_index_fk_city_ref_users            |                       9  
+       55 | PARTITION europe_west OF INDEX movr.public.rides@rides_auto_index_fk_vehicle_city_ref_vehicles | NULL       | movr          | rides      | rides_auto_index_fk_vehicle_city_ref_vehicles |                       9  
+       55 | PARTITION europe_west OF INDEX movr.public.rides@rides_auto_index_fk_vehicle_city_ref_vehicles | NULL       | movr          | rides      | rides_auto_index_fk_vehicle_city_ref_vehicles |                       9  
+       55 | PARTITION europe_west OF INDEX movr.public.rides@rides_auto_index_fk_vehicle_city_ref_vehicles | NULL       | movr          | rides      | rides_auto_index_fk_vehicle_city_ref_vehicles |                       9  
+(30 rows)
+~~~
+
+### Find out which databases and tables have at-risk ranges
+
+By default, this geo-distributed demo cluster will not have any at-risk ranges. A range is considered "at-risk" if its data could be lost in case of a node failure.
+
+To force the cluster into a state where some ranges are at risk, issue the following statement, which tells it to store 9 copies of each range underlying the `rides` table (by default [it stores 3](configure-replication-zones.html#num_replicas)).
+
+{% include copy-clipboard.html %}
+~~~ sql
+ALTER TABLE rides CONFIGURE ZONE USING num_replicas=9;
+~~~
+
+Once the statement above runs, the cluster will rebalance so that it's storing 9 replicas of each range underlying the `rides` table. This puts that data at risk since you are storing 9 copies of the table's data on only 9 nodes.  If a node fails, you will be left with 8 replicas, and there will be no additional nodes to upreplicate the range to to fulfill the `num_replicas=9` requirement.
```

Comment:
> If a node fails, you will be left with 8 replicas, and there will be no additional nodes to upreplicate the range to to fulfill the `num_replicas=9` requirement.

I don't think this is right. There might be a misunderstanding about what makes a locality be critical. It's not about whether the replication factor could still be met. In a 3 localities setup, 3 nodes per locality, and everything replicated 9x, no locality is critical. You can lose any one locality without losing quorum for any ranges.